delta/plugins/blazegraph/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/blazegraph/routes/ResponseToSparql.scala

```diff
@@ -0,0 +1,43 @@
+package ch.epfl.bluebrain.nexus.delta.plugins.blazegraph.routes
+
+import akka.http.scaladsl.model.StatusCodes.OK
+import akka.http.scaladsl.server.Route
+import ch.epfl.bluebrain.nexus.delta.plugins.blazegraph.client.SparqlResults
+import ch.epfl.bluebrain.nexus.delta.rdf.RdfMediaTypes
+import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.context.RemoteContextResolution
+import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.encoder.JsonLdEncoder
+import ch.epfl.bluebrain.nexus.delta.rdf.utils.JsonKeyOrdering
+import ch.epfl.bluebrain.nexus.delta.sdk.directives.Response.Complete
+import ch.epfl.bluebrain.nexus.delta.sdk.directives.{Response, ResponseToJsonType}
+import ch.epfl.bluebrain.nexus.delta.sdk.marshalling.{HttpResponseFields, RdfMarshalling}
+import io.circe.Json
+import io.circe.syntax._
+import monix.bio.{IO, UIO}
+import monix.execution.Scheduler
+
+sealed trait ResponseToSparql {
+  def apply(): Route
+}
+
+object ResponseToSparql extends SparqlValueInstances {
```

Comment:
this extends SparqlValueInstance is not doing anything here, since in emit you have an instance of `ResponseToJsonType`